OpenSim/Actuators/VandenBogert2011Muscle.h

```diff
@@ -0,0 +1,568 @@
+* See http://opensim.stanford.edu and the NOTICE file for more information.  *
+* OpenSim is developed at Stanford University and supported by the US        *
+* National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+* through the Warrior Web program.                                           *
+*                                                                            *
+* Copyright (c) 2016 Stanford University and the Authors                     *
+* Author(s): Brad Humphreys, Chris Dembia, Antoine van den Bogert            *
+*                                                                            *
+* Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+* not use this file except in compliance with the License. You may obtain a  *
+* copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+*                                                                            *
+* Unless required by applicable law or agreed to in writing, software        *
+* distributed under the License is distributed on an "AS IS" BASIS,          *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+* See the License for the specific language governing permissions and        *
+        * limitations under the License.                                     *
+* -------------------------------------------------------------------------- */
+
+#include <OpenSim/Actuators/osimActuatorsDLL.h>
+#include <simbody/internal/common.h>
+
+/*
+The parent class, Muscle.h, provides
+    1. max_isometric_force
+    2. optimal_fiber_length
+    3. tendon_slack_length
+    4. pennation_angle_at_optimal
+    5. max_contraction_velocity
+*/
+#include <OpenSim/Simulation/Model/Muscle.h>
+
+/*
+#ifdef SWIG
+#ifdef OSIMACTUATORS_API
+        #undef OSIMACTUATORS_API
+        #define OSIMACTUATORS_API
+    #endif
+#endif
+*/
+
+
+namespace OpenSim {
+/**
+This class implements a muscle with implicit dynamics capabilities as described
+ in van den Bogert et al. \(2011).
+ */
+    //TODO:  Add detailed muscle discription
+
+// Derive new class from Muscle.h
+class OSIMACTUATORS_API VandenBogert2011Muscle : public Muscle {
+//TODO Do I need: class OSIMACTUATORS_API VandenBogert2011Muscle......
+        OpenSim_DECLARE_CONCRETE_OBJECT(VandenBogert2011Muscle, Muscle);
+
+public:
+//=============================================================================
+// PROPERTIES
+//=============================================================================
+
+        //OpenSim_DECLARE_PROPERTY(max_isometric_force, double,
+        //  F_{max} (N) "Maximum isometric force that the fibers can generate");
+        // Comes from parent class, Muscle.h
+
+        OpenSim_DECLARE_PROPERTY(fMaxTendonStrain, double,
+        "tendon strain at maximum isometric muscle force");
+        //u_{max} (dimensionless) strain in the series elastic element at load
+        // of maxIsometricForce
```

Comment:
Okay thank you for clarifying. It would be good to at least mention which properties are related to curve shape (as opposed to ones like activation time constant, pennation at optimal fiber length, etc.).